ui/wallets_page.go

```diff
@@ -31,7 +33,7 @@ func (win *Window) WalletsPage() {
 								win.theme.Body1(acct.Name).Layout(win.gtx)
 							}),
 							layout.Rigid(func() {
-								win.theme.Body1(acct.TotalBalance).Layout(win.gtx)
+								win.theme.Body1(strconv.Itoa(int(acct.TotalBalance))).Layout(win.gtx)
```

Comment:
You seem to have chosen your version of the code over that on master when you rebased.